src/pass/plan_memory.cc

```diff
@@ -197,28 +166,38 @@ Graph PlanMemory(Graph ret) {
         uint32_t eid_in = idx.entry_id(inode.inputs[kv.first]);
         if (ref_count[eid_in] == 1 &&
             ref_count[eid_out] != 0 &&
-            storage[eid_out] == GraphAllocator::kBadStorageID &&
-            storage[eid_in] != GraphAllocator::kBadStorageID &&
+            (*storage)[eid_out] == GraphAllocator::kBadStorageID &&
+            (*storage)[eid_in] != GraphAllocator::kBadStorageID &&
             shape_vec[eid_out].Size() == shape_vec[eid_in].Size() &&
             dtype_vec[eid_out] == dtype_vec[eid_in]) {
           // inplace optimization
-          storage[eid_out] = storage[eid_in];
+          (*storage)[eid_out] = (*storage)[eid_in];
```

Comment:
`src/pass/plan_memory.cc:144:  Is this a non-const reference? If so, make const or use a pointer: StorageVector& storage  [runtime/references] [2]`
This is the error message when I pass as reference